pyspecProcScripts/first_level/fake_data.py

```diff
@@ -0,0 +1,95 @@
+from .fl_dummy import fl_dummy
+from sympy import Symbol as sympy_symbol
+from sympy import lambdify
+from pyspecdata import nddata, ndshape
+import numpy as np
+from numpy.random import normal
+def fake_data(
+        expression,
+        axis_coords,
+        signal_pathway,
+        direct = 't2',
+        SD_sigma = [0.05,0.003],
+        SD_amp = [1,5],
+        scale=100,
+        fake_data_noise_std = 1.0,
+        fl=fl_dummy):
+    """Generate fake data subject to noise and frequency variation.
+    Parameters
+    ==========
+    expression: sympy expression
+        Gives the functional form of the data.
+    axis_coords: OrderedDict
+        Gives nddata objects providing all the axis coordinates.
+        **Very importantly**, these must be listed in the loop nesting order
+        (outside in) in which they occur in the pulse program,
+        or the frequency drift will not be modeled correctly.
+
+        To enable *simulating echo-like data*, you can specify a direct axis
+        that starts at a negative number.
+        If you do this, the beginning of the axis will be re-set to 0 before returning.
+    SD_sigma: list of floats
+        Gives the Gaussian σ for the spectral density of the
+        frequency variation.
+        Typically, there are more than one gaussian terms used to
+        create the spectral density.
+
+        A small σ gives rise to slow fluctuations while a large σ
```

Comment:
is fluctuations the rainbowy phasing we see?

Reply:
No, that comes from the fact that the echo is not centered at zero￼. 

The frequency draft results in the fact that the center of the signal is wiggling back and forward￼